src/main/webapp/WEB-INF/jsp/merchant.jsp

```diff
@@ -40,23 +49,24 @@
 							class="glyphicon glyphicon-chevron-right"></span> Debit</a></li>
 					<li id="transfer"><a href="transfer"><span
 							class="glyphicon glyphicon-chevron-right"></span> Transfer</a></li>
-					<li id="addAccount"><a href="regular"><span
+					<li id="addAccount"><a href="#"><span
 							class="glyphicon glyphicon-chevron-right"></span> Add Account</a></li>
-					<li id="transactions"><a href="searchTransaction"><span
+					<li id="transactions"><a href="externalsearchtrans"><span
 							class="glyphicon glyphicon-chevron-right"></span> Transactions</a></li>
-					<li id="notifications"><a href="searchTransaction"><span
+					<li id="bankStatement"><a href="bankStatement"><span
+							class="glyphicon glyphicon-chevron-right"></span> Bank Statement</a></li>
+					<li id="notifications"><a href="notifications"><span
 							class="glyphicon glyphicon-chevron-right"></span> Notifications</a></li>
 					<li id="createRequest"><a href="#"><span
 							class="glyphicon glyphicon-chevron-right"></span> Create Request</a></li>
-					<li id="logout"><a href="#"><span
```

Comment:
Why you remove this? I did this form should be submitted..This should be inside form else logout will not work